src/libraries/System.Private.Xml/src/System/Xml/Serialization/XmlSerializationILGen.cs

```diff
@@ -67,7 +69,7 @@ internal ModuleBuilder ModuleBuilder
         private static readonly Dictionary<string, Regex> s_regexs = new Dictionary<string, Regex>();
         internal static Regex NewRegex(string pattern)
```

Comment:
Should the return type be `Regex?` here?

Reply:
why? it will be initialized if it is not in the dictionary, also no `!` anywhere and compiler doesn't complain 😄

Reply:
Oh, s_regexs accepts only non-null values. I got thrown off by the `Regex? regex` declaration. I agree, this is correct.